dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -0,0 +1,332 @@
+import { css } from '@emotion/react';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign, ArticleDisplay } from '@guardian/libs';
+import { from, space, until } from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import { interactiveLegacyClasses } from '../layouts/lib/interactiveLegacyStyling';
+import { getSoleContributor } from '../lib/byline';
+import { palette as themePalette } from '../palette';
+import type { Branding as BrandingType } from '../types/branding';
+import type { TagType } from '../types/tag';
+import { shouldShowAvatar, shouldShowContributor } from './ArticleMeta';
+import { Avatar } from './Avatar';
+import { Branding } from './Branding.importable';
+import { CommentCount } from './CommentCount.importable';
+import { Contributor } from './Contributor';
+import { Dateline } from './Dateline';
+import { FollowWrapper } from './FollowWrapper.importable';
+import { Island } from './Island';
+import { SendAMessage } from './SendAMessage.importable';
+
+type Props = {
+	format: ArticleFormat;
+	pageId: string;
+	webTitle: string;
+	byline?: string;
+	tags: TagType[];
+	primaryDateline: string;
+	secondaryDateline: string;
+	branding?: BrandingType;
+	discussionApiUrl: string;
+	shortUrlId: string;
+	isCommentable: boolean;
+	ajaxUrl: string;
+	messageUs?: MessageUs;
+};
+
+const metaGridContainer = css`
+	display: grid;
+	grid-template-rows: auto auto auto auto auto;
+	grid-template-columns: 10px auto 1fr auto 10px;
+	grid-template-areas:
+		' . . . . .'
+		'. avatar byline comment-count .'
+		'. avatar byline comment-count .'
+		' . . . . .'
+		'. dateline dateline dateline .';
```

Comment:
Might be slightly easier to parse visually?

```suggestion
		'. .        .        .             .'
		'. avatar   byline   comment-count .'
		'. avatar   byline   comment-count .'
		'. .        .        .             .'
		'. dateline dateline dateline      .';
```